specs/api.spec.js

```diff
@@ -1,78 +1,141 @@
-      method: 'POST',
-      headers: { 'Content-Type': 'application/json' },
-      body: JSON.stringify({
-        userName: config.login_correct,
-        password: config.password_correct
-      })
+describe('Generate token tests', () => {
+  test('for success generate token', async () => {
+    const response = await UserService.generate({
+      userName: testUserName,
+      password: testUserPassword
     })
-    const data = await response.json()
     expect(response.status).toBe(200)
-    expect(data.status).toBe('Success')
-    expect(data.token).toBeTruthy()
+    expect(response.data.status).toBe('Success')
+    expect(response.data.token).toBeTruthy()
   })
   test('for unsuccessful generate token', async () => {
-    const response = await fetch(config.baseURL + '/Account/v1/GenerateToken', {
-      method: 'POST',
-      headers: { 'Content-Type': 'application/json' },
-      body: JSON.stringify({
-        userName: config.login_correct,
-        password: config.password_incorrect
-      })
+    const response = await UserService.generate({
+      userName: testUserName,
+      password: config.password_incorrect
+    })
+    expect(response.status).toBe(200)
+    expect(response.data.status).toBe('Failed')
+    expect(response.data.token).toBeNull()
+  })
+})
+
+describe('Auth user tests', () => {
+  test('success user authorized', async () => {
+    const response = await AuthService.login({
+      userName: testUserName,
+      password: testUserPassword
+    })
+    expect(response.status).toBe(200)
+    expect(response.data).toBe(true)
+  })
+  test('incorrect password', async () => {
+    const response = await AuthService.login({
+      userName: testUserName,
+      password: config.password_incorrect
+    })
+    expect(response.status).toBe(400) //баг в api - 400 отдаётся только на пустой пароль, а должен на некорректный.
+    expect(response.data.message).toBeTruthy()
+  })
+  test('user not found', async () => {
+    const response = await AuthService.login({
+      userName: 'user',
+      password: testUserPassword
     })
-    const data = await response.json()
+    expect(response.status).toBe(404)
+    expect(response.data.message).toBe('User not found!')
+  })
+  test('username not specified', async () => {
+    const response = await AuthService.login({
+      userName: null,
+      password: testUserPassword
+    })
+    expect(response.status).toBe(400)
+    expect(response.data.message).toBe('UserName and Password required.')
+  })
+  test('password not specified', async () => {
+    const response = await AuthService.login({
+      userName: testUserName,
+      password: null
+    })
+    expect(response.status).toBe(400)
+    expect(response.data.message).toBe('UserName and Password required.')
+  })
+})
+
+describe('Get info about user tests', () => {
+  test('for success get info', async () => {
+    const response = await UserService.get(testUserId, token)
```

Comment:
Вы не правильно передаёте параметры в функцию:

```suggestion
    const response = await UserService.get({ userID: testUserId, token })
```